app/src/main/java/com/github/palFinderTeam/palfinder/meetups/activities/MeetUpCreation.kt

```diff
@@ -406,5 +419,7 @@ class MeetUpCreation : Fragment(R.layout.activity_meet_up_creation_new), IconDia
 
     companion object {
         private const val ICON_DIALOG_TAG = "icon-dialog"
+        private const val EASTER_NAME = "TARTiNE"
```

Comment:
do they really need to be in the companion object, and an easter egg should be something findable

Reply:
Okay so maybe I'll change it to all caps

Reply:
I thought the companion object was the best place to store, it seemed better than in the strings

Reply:
As you want, it's closer to the code where it is used I guess...
+ 192.168.0.1